delphi/translators/for2py/pyTranslate.py

```diff
@@ -1142,6 +1143,92 @@ def get_python_source(self):
 
         return "".join(self.pyStrings)
 
+    """
+        This function checks the type of a variable and returns the appropriate python syntax type name
```

Comment:
This line should be less than 80 characters in length.  Similarly for other docstrings.